pkg/skaffold/deploy/resource/deployment.go

```diff
@@ -165,15 +175,24 @@ func (d *Deployment) ReportSinceLastUpdated() string {
 	for _, p := range d.pods {
 		if s := p.ActionableError().Message; s != "" {
 			result.WriteString(fmt.Sprintf("%s %s %s: %s\n", tab, tabHeader, p, s))
-			for _, l := range p.Logs() {
-				result.WriteString(fmt.Sprintf("%s\n", l))
+			// if, is muted then write container logs to file and last 3 lines to
+			// result.
+			out, writeTrimLines, err := withLogFile(p.Name(), &result, p.Logs(), isMuted)
+			if err != nil {
+				logrus.Debugf("could not create log file %v", err)
+			}
+			trimLines := []string{}
+			for i, l := range p.Logs() {
+				formattedLine := fmt.Sprintf("%s %s > %s\n", tab, tab, strings.TrimSuffix(l, "\n"))
+				if isMuted && i >= len(p.Logs())-3 {
+					trimLines = append(trimLines, formattedLine)
```

Comment:
can we just show the message here, and omit the other info? the JSON output is a bit much for the abbreviated log lines, especially since we're just trying to show the error **and** we're capturing detailed logs to a file.

Reply:
@nkubala the json output is actually coming from the nodejs container from guest book sample.
```
> [nodejs-guestbook-mongodb-579dc8b69d-rdlpv mongo] {"t":{"$date":"2020-08-18T11:53:41.140+00:00"},"s":"I",  "c":"-",        "id":20520,   "ctx":"initandlisten","msg":"Stopping further Flow Control ticket acquisitions."}
      > [nodejs-guestbook-mongodb-579dc8b69d-rdlpv mongo] {"t":{"$date":"2020-08-18T11:53:41.141+00:00"},"s":"I",  "c":"CONTROL",  "id":20565,   "ctx":"initandlisten","msg":"Now exiting"}
      > [nodejs-guestbook-mongodb-579dc8b69d-rdlpv mongo] {"t":{"$date":"2020-08-18T11:53:41.141+00:00"},"s":"I",  "c":"CONTROL",  "id":23138,   "ctx":"initandlisten","msg":"Shutting down","attr":{"exitCode":100}}
```

If you look at the tests, the output is 
```
 - test-ns:deployment/test: container terminated with exit code 11
    - test:pod/foo: container terminated with exit code 11
      > [pod container] Waiting for mongodb to start...
      > [pod container] Waiting for connection for 2 sec
      > [pod container] Terminating with exit code 11
```

Reply:
ahhh got it. pretty gross output on their side :)